config.yaml

```diff
@@ -0,0 +1,35 @@
+linkerdVersion: stable-2.8.0
+externalIssuer: false
+controlPlane:
+    # namespace: l5d-conformance
+    config:   
+        ha: false
+        flags:
+            - "--controller-log-level"
+            - "debug"
+            - "--proxy-log-level"
+            - "warn,linkerd2_proxy=debug"
+      # addOns:
+      #     tracing:
+      #         enabled: true
+      #         collector:
+      #             resources:
+      #                 cpu:
+      #                     limit: 100m
+      #                     request: 10m
+      #                 memory:
+      #                     limit: 100Mi
+      #                     request: 50Mi
+lifecycle:
+    skip: false
+    reinstall: false 
+    uninstall: true
+    # upgradeFromVersion: stable-2.7.0
+inject:
+    skip: false
+    clean: true
+ingress:
+    skip: false
+    config:
+        controllers:
+            - nginx
```

Comment:
I think we should nest these test case configuration, under a parent variable named `testCase`. This will separate the test case configuration from the general meta configuration like Linkerd version, binary path,

E.g.,

```yaml
testCase:
  lifecycle:
    skip: false
    reinstall: false 
    uninstall: true   
  inject:
    skip: false
    clean: true
  ingress:
    skip: false
    config:
      controllers:
       - nginx
```